WordPressCom-Stats-iOS/StatsEphemory.m

```diff
@@ -13,7 +13,7 @@ @implementation EphemoryContainer
 
 @interface StatsEphemory ()
 
-@property (nonatomic, strong) NSCache *cache;
+@property (nonatomic, strong) NSMutableDictionary *cache;
```

Comment:
If you do go that way (NSDictionary persistance), take a look at [this helper[(https://github.com/Simperium/simperium-ios/blob/develop/Simperium/SPPersistentMutableDictionary.h), might save you some time.

That guy is what Simperium uses, internally, to persist pending changes.